beetsplug/lyrics.py

```diff
@@ -373,7 +373,13 @@ def lyrics_from_song_api_path(self, song_api_path):
 
         # At least Genius is nice and has a tag called 'lyrics'!
         # Updated css where the lyrics are based in HTML.
-        lyrics = html.find("div", class_="lyrics").get_text()
+        try:
+            lyrics = html.find("div", class_="lyrics").get_text()
+        except AttributeError as exc:
```

Comment:
Cool! I think this might be a little clearer (and not require the code-like reasoning that currently appears in a comment) if we instead check specifically for None instead of generically for a missing attribute:

```py
lyrics_div = html.find(...)
if lyrics_div is None:
    self._log...
    return None
lyrics = lyrics_div.get_text()
```

What do you think about that minor restructuring?

Reply:
Seems like it should have the same functionality, I'll rewrite it and test it again to make sure! Checking for None in general seems like a better failsafe, I think I was just using try-except because that's what the other error checking seemed to be doing.